x-pack/plugins/lens/public/indexpattern_datasource/field_item.test.tsx

```diff
@@ -227,31 +227,14 @@ describe('IndexPattern Field Item', () => {
     clickField(wrapper, 'bytes');
 
     expect(core.http.post).toHaveBeenCalledTimes(2);
-    expect(core.http.post).toHaveBeenLastCalledWith(`/api/lens/index_stats/1/field`, {
-      body: JSON.stringify({
-        dslQuery: {
-          bool: {
-            must: [],
-            filter: [
-              {
-                bool: {
-                  should: [{ match_phrase: { 'geo.src': 'US' } }],
-                  minimum_should_match: 1,
-                },
-              },
-              {
-                match: { phrase: { 'geo.dest': 'US' } },
-              },
-            ],
-            should: [],
-            must_not: [],
-          },
+    expect(core.http.post.mock.calls[0]).toMatchInlineSnapshot(`
```

Comment:
Snapshot tests are weaker than exact match tests, are you sure that changing this to a snapshot is testing the right thing?

Reply:
I'm wondering the same thing.

I am not opposed to it from our side, but one big con we have seen within `security_solutions` is that people tend to update the jest snapshots very quickly as a knee jerk reaction and then they will put the burden on the code reviewer to find where and if a mistake is made.

Typically if they have to at least update the objects, then we at least force them to see the changes by updating them manually vs. automatically. After a few code reviews where there were bugs and no one noticed, the snapshots began down trending from our side.

It's faster (true) if this output generator changes again which is a pro though. I do admit that.

I can approve this as-is with the inline snapshots unless someone else is strongly apposed from `security_solutions` and chimes in.